index.js

```diff
@@ -2,6 +2,8 @@ const fs = require('fs');
 const path = require('path');
 const connectMultiparty = require('connect-multiparty');
 const fields = require('./lib/fields');
+const recaptcha = require('./lib/recaptcha');
+const process = require('./lib/process');
```

Comment:
I wouldn't call a variable `process` because code in the module might need to access the `process` global at some point.